src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -298,7 +378,7 @@ class ActivityFeed extends React.Component<Props, State> {
                     }}
                     className="bcs-activity-feed-items-container"
                 >
-                    {isLoading && (
+                    {(isLoading || isReplyDataLoading) && (
```

Comment:
The combination of loading state here indicates again that this should probably occur when the feed items are being fetched rather than separately. If not, though, should this move into the `hasLoaded` method?